knowledge-base/infracost/actions/setup/action-security.yml

```diff
@@ -0,0 +1,10 @@
+name: "Setup Infracost"
+github-token:
+  action-input:
+    input: github-token
+    is-default: true
+  permissions:
+    contents: read
+    contents-reason: to get list of Infracost cli versions #Checkout: https://github.com/infracost/actions/blob/master/setup/action.yml#L22
```

Comment:
@h0x0er I think it does not need contents: read because it is pulling from a different repo. GitHub token perm is for the current repo. Can you please change this to no perms?